src/picongpu/include/plugins/radiation/Radiation.kernel

```diff
@@ -234,145 +238,143 @@ void kernelRadiationParticles(ParBox pb,
                  * LATER: can this be optimized?
                  */
                 int saveParticleAt = -1;
-                PMACC_AUTO(par,(*frame)[linearThreadIdx]);
 
-                /* if radiation is not calculated for all particles
-                 * but not via the gamma filter, check which particles
-                 * have to be used for radiation calculation
-                 */
+                /* if particle is not accelerated we skip all calculations */
+                if( particle_momentumNow != particle_momentumOld )
+                {
+                    /* if radiation is not calculated for all particles
```

Comment:
this double "not" (not calculated for all "but not via") is pretty confusing. let us try to rewrite that with less negations.